### PR TITLE
[INLONG-7256][Manager] Add a check for Pulsar service URL

### LINKED ISSUE #7256: [Bug][Manager] The test connection address is wrong, but it shows success
### What happened

image

### What you expected to happen

The test connection should check service urls

### How to reproduce

Add pulsar cluster

### Environment

_No response_

### InLong version

1.5.0

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request
- Fixes #7256 

### Motivation

Add a check for serverUrl
### Modifications

Add a check for serverUrl

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/PulsarClusterOperator.java [hunk 1]
```diff
@@ -44,6 +50,8 @@ public class PulsarClusterOperator extends AbstractClusterOperator {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PulsarClusterOperator.class);
 
+    private static final String SERVER_URL_PREFIX = "pulsar://";
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/PulsarClusterOperator.java:53]: It should be `service URL`.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/PulsarClusterOperator.java [hunk 2]
```diff
@@ -87,17 +95,29 @@ protected void setTargetEntity(ClusterRequest request, InlongClusterEntity targe
...
-    @Override
     public Boolean testConnection(ClusterRequest request) {
         PulsarClusterRequest pulsarRequest = (PulsarClusterRequest) request;
         PulsarClusterInfo pulsarInfo = new PulsarClusterInfo();
         CommonBeanUtils.copyProperties(pulsarRequest, pulsarInfo);
         try (PulsarAdmin ignored = PulsarUtils.getPulsarAdmin(pulsarInfo)) {
-            LOGGER.debug("pulsar connection not null - connection success for adminUrl={}", pulsarInfo.getAdminUrl());
+            ignored.tenants().getTenants();
+            String serviceUrl = pulsarInfo.getUrl();
+            Preconditions.expectNotNull(serviceUrl, "ServerUrl is empty");
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/PulsarClusterOperator.java:105]: Please change all `serverUrl` to `serviceUrl` for Apache Pulsar.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/PulsarClusterOperator.java [hunk 3]
```diff
@@ -87,17 +95,29 @@ protected void setTargetEntity(ClusterRequest request, InlongClusterEntity targe
         }
     }
 
-    @Override
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/PulsarClusterOperator.java:90]: Why remove it?

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/PulsarClusterOperator.java [hunk 4]
```diff
@@ -87,17 +95,29 @@ protected void setTargetEntity(ClusterRequest request, InlongClusterEntity targe
...
     public Boolean testConnection(ClusterRequest request) {
         PulsarClusterRequest pulsarRequest = (PulsarClusterRequest) request;
         PulsarClusterInfo pulsarInfo = new PulsarClusterInfo();
         CommonBeanUtils.copyProperties(pulsarRequest, pulsarInfo);
         try (PulsarAdmin ignored = PulsarUtils.getPulsarAdmin(pulsarInfo)) {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/PulsarClusterOperator.java:102]: `ignored` is weired too.

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by leosanqing — APPROVED** [review]

**Pull request merged**